src/jdk.crypto.cryptoki/unix/native/libj2pkcs11/p11_md.c

```diff
@@ -91,7 +91,7 @@ JNIEXPORT jobject JNICALL Java_sun_security_pkcs11_wrapper_PKCS11_connect
     jobject globalPKCS11ImplementationReference;
     char *systemErrorMessage;
     char *exceptionMessage;
-    const char *getFunctionListStr = NULL;
+    const char *getFunctionListStr = "C_GetFunctionList";
```

Comment:
If this value ever gets used by ReleaseStringUTFChars, the failure will be spectacular 🍿

Reply:
We do have checks for jGetFunctionList != NULL before calling ReleaseStringUTFChars() with it. So, this shouldn't be an issue?

Reply:
this will be an issue if `jGetFunctionList != NULL` and `dlopen` fails, resulting in `goto cleanup` before `getFunctionListStr` is reassigned.

Reply:
Hmm, ok, I will change it. Thanks~